0800-appendix-195943-stock.Rmd

```diff
@@ -60,40 +50,19 @@ fpr::fpr_table_wshd_sum(site_id = my_site) %>%
 
 <br>
 
-although no records of the falls have yet been recorded in
-@moe2020ProvincialObstacles, @redden_jedrzejczyk1997FishFish report that
-a 15m high rock falls is located at 7.8km upstream from the confluence
-with the upper Bulkley River and represents the upstream limit of fish
-distribution in the watershed. They also report excellent and good
-rearing potential in upper reaches of the stream located from 5.1km to
-7.8km upstream of the confluence.
-
-<br>
-
-@mccarthy_fernando20152015Inventory report that the culvert has received
-past efforts to facilitate backwatering including rock lines to
-establish backwater pools downstream of the culvert. They also report
-that the works were not functioning correctly.
-@smith2018AssessingBarriers visited the site in 2017, subjectively rated
-the crossing as a potential barrier and ranked the crossing as a low
-priority for follow up. @irvine2021BulkleyRiver noted that a design was
-being drafted for the Highway 16 crossing by Ministry of Transportation
-and Infrastructure in 2021.
+Dogwood Environmental Consulting reports that they are working on a grazing/fuel reduction pilot project in collaboration with BC Wildfire, the Ministry of Agriculture, and BC Cattleman’s Association. Dogwood is working with grazing lease holders and farmers to identify agricultural areas in need of enhancement. Stock Creek is included in one of these target areas, due to its clear signs of cattle use and grazing, leading to significant impacts on stream habitat. Plans to mitigate these impacts and prevent cattle from accessing stream banks are ongoing. The project study area is located upstream of the assessed Highway 16 crossing (pers. comm Jenn Atkins, 2022). 
 
 <br>
 
 PSCIS stream crossing `r as.character(my_site)` was ranked as a high
 priority for follow up by Canadian Wildlife Federation because of
-significant amounts of habitat modelled as upstream of the crossing.
-Report comments indicate that the site was given a moderate ranking for
-follow up in @irvine2018AnalysisPriority as the site appeared dry in
```

Comment:
why remove this? Seems relevant to me. You can obviously cntrl-F to search that document to find that comment